tools/import_validate.py

```diff
@@ -34,19 +34,41 @@ def blank_import(data):
     if blanks[-1] == num - 1:
         blanks.pop(-1)
 
+def check_imports(data):
+    # We only tolerate blank lines between first and third party imports, as well as blank (_) imports
+    line = data.get('line', '')
+    import_state = data.get('import_state', TPIMPORTS)
+    # ignore . imports, but recognize 3rd party imports of the form github.com/user/proj
+    if not line.startswith('.') and '.' in line and import_state == FPIMPORTS:
+        data["import_state"] = TPIMPORTS
+        check_blanks(data)
```

Comment:
for some reason I always thought you had to define the function before using it in python. I guess not 😄